server/app/lib/actions/fusor/deployment/openstack/ssh_command.rb

```diff
@@ -36,7 +36,7 @@ def run
       client.on_failure(lambda { ssh_command_failed })
       client.execute(input[:cmd])
 
-      ::Fusor.log.debug "================ Leaving SshCommand run method ===================="
+      ::Fusor.log.info "================ Leaving SshCommand run method ===================="
```

Comment:
These statements are useless at info level because we are created 5-7 SshCommands in openstack/deploy.rb
https://github.com/fusor/fusor/blob/master/server/app/lib/actions/fusor/deployment/openstack/deploy.rb#L31-L46

All we'll see in the log is a bunch of lines like this:
```
================ SshCommand run method ====================
================ Leaving SshCommand run method ====================
================ SshCommand run method ====================
================ Leaving SshCommand run method ====================
================ SshCommand run method ====================
================ Leaving SshCommand run method ====================
================ SshCommand run method ====================
================ Leaving SshCommand run method ====================
```

I would leave these as debug, and add an info above the ```execute```

```
::Fusor.log.info "SshCommand: executing #{input[:cmd]}"
client.execute(input[:cmd])
```

That way we can see what command was about to run in case of errors.